examples/spawn-and-move/src/actions.cairo

```diff
@@ -1,15 +1,15 @@
 use dojo_examples::models::{Position, Vec2};
 
-#[starknet::interface]
-trait IActions<TContractState> {
-    fn spawn(self: @TContractState);
-    fn move(self: @TContractState, direction: dojo_examples::models::Direction);
+#[dojo::interface]
+trait IActions {
+    fn spawn();
+    fn move(direction: dojo_examples::models::Direction);
```

Comment:
To be consistent, could we import the Direction and use it as `Position` and `Vec2` while we're editing the code here?
```suggestion
    fn move(direction: Direction);
```